recipe/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
   sha256: d9dc4b3318f310e34c82951ea5d6683f67bed7def4b259fafbfe4f1beb1d8e5f
 
 build:
```

Comment:
Do we not need a script section here because the `build.sh` and `bld.bat` exist in the recipe?

Reply:
Correct!  If it isn't there conda-build will look for those files by default.